iidm/iidm-api/src/main/java/com/powsybl/iidm/network/TapChanger.java

```diff
@@ -221,4 +221,8 @@ default Map<Integer, S> getAllSteps() {
         }
         return steps;
     }
+
+    default void setTapPositiontoSolvedTapPosition() {
+        this.findSolvedTapPosition().ifPresent(this::setTapPosition);
```

Comment:
I wonder if we shouldn't also unset the value if the solved value is absent?
But this may downgrade the network to EQUIPEMENT level (or throw an exception), for instance if a model doesn't affect a solvedTap to a disconnected transformer.